<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winDoc</class>
 <widget class="QDialog" name="winDoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Détail du document</string>
  </property>
  <widget class="QLabel" name="labelPicture">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../ressources/ressources.qrc">:/winPictures/doc_64.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelNameDoc">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDesc">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;Détail du document&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNameDoc">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditDescDoc">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtnOk">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../ressources/ressources.qrc"/>
 </resources>
 <connections/>
</ui>
